packages/core/src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,107 @@
+import AriaModal from 'react-aria-modal';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const Dialog = function(props) {
+  const {
+    actions,
+    children,
+    escapeExitDisabled,
+    onExit,
+    title,
+    ...modalProps
+  } = props;
+
+  return (
+    <AriaModal
+      dialogClass="ds-c-dialog"
+      escapeExits={!escapeExitDisabled}
+      includeDefaultStyles={false}
+      onExit={onExit}
+      titleId="dialog-title"
+      underlayClass="ds-c-dialog-wrap"
+      {...modalProps}
+    >
+      <header className="ds-c-dialog__header">
```

Comment:
Let's add a `role="banner"` to this header element, for backward compatibility.

Reply:
@1Copenut Is it better to add `role="banner"` or change this element to a normal `div`?